Automattic/simplenote-ios
# Charlie/drop sustainer

### Fix
We are dropping the Sustainer plan from Simplenote.  This PR removes the Sustainer banner in the tags list, the sustainer banner in the settings view, and the alert controller that guides users to upgrading.  While looking at this we decided that the store and header banner tools may be useful in the future so I have preserved these tools and updated the naming of `SustainerView` to `BannerView` to make it more usable in the future.

### Test
***(Required)*** List the steps to test the behavior.  For example:
1. Launch Simplenote iOS and confirm that in the tags list you do not see the sustainer banner.  
2. Open the settings view and confirm there is no sustainer banner in the settings view
3. Make sure you can not access the alert controller that permits purchasing sustainer
4. Make sure both the tags and settings views respond to the device orientation changes.

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
If the changes should not be included in release notes, add a statement to this section. For example:

> These changes do not require release notes.
